cmasher/tests/test_utils.py

```diff
@@ -65,6 +67,105 @@ def _MPL38_colormap_eq(cmap, other) -> bool:
 
 
 # %% PYTEST CLASSES AND FUNCTIONS
+# Pytest class for combine_cmaps
+class Test_combine_cmaps:
+    # Test if multiple Colormaps or colormap names can be combined
+    @pytest.mark.parametrize(
+        "cmaps, nodes",
+        [
+            (["Blues", "Oranges", "Greens"], [0.25, 0.75]),
+            (["Blues", "Oranges", "Greens"], np.array([0.25, 0.75])),
+            (
+                [
+                    mpl.colormaps["Blues"],
+                    mpl.colormaps["Oranges"],
+                    mpl.colormaps["Greens"],
+                ],
+                [0.25, 0.75],
+            ),
+        ],
+    )
+    def test_combine_cmaps(self, cmaps, nodes):
+        combined_cmap = combine_cmaps(cmaps, nodes=nodes, n_rgb_levels=256)
+        blues_cmap = mpl.colormaps["Blues"]
+        oranges_cmap = mpl.colormaps["Oranges"]
+        greens_cmap = mpl.colormaps["Greens"]
+
+        assert np.allclose(combined_cmap(0.0), blues_cmap(0))
+        assert np.allclose(combined_cmap(0.25), oranges_cmap(0))
+        assert np.allclose(combined_cmap(0.75), greens_cmap(0))
+        assert np.allclose(combined_cmap(1.0), greens_cmap(255))
+
+        assert combined_cmap.N == 256
+
+    # Test if combining a single colormap raises an error
+    def test_single_cmap(self):
+        with pytest.raises(ValueError, match="Cannot combine a single colormap."):
+            combine_cmaps(cmaps=["Blues"])
```

Comment:
Together with my suggestion to also error if we have 0 inputs (this will require a formatting step)
```suggestion
    # Test if combining less than 2 colormaps raises an error
    @pytest.mark.parametrize("cmaps", [pytest.param([], id="no_cmap"), pytest.param("Blues", id="single_cmap"), pytest.param("fake_name", id="fake_cmap_name")]) 
    def test_single_cmap(self):
        with pytest.raises(ValueError, match="Cannot combine a single colormap."):
            combine_cmaps(cmaps=["Blues"])
```

Reply:
`pytest.param("fake_name", id="fake_cmap_name")` seems to trigger a different error here, and is unrelated to this unit test? This should be testing the number of cmaps, instead of the validity of cmaps?

Reply:
Well, the idea is that passing a single name *should* trigger the same error, regardless if the name is valid or not, because we validate the number of inputs before we check their validity. So if the test is failing, that's probably indicative of a bug (and that's what these tests are for), so please include it (... and fix the bug :))

Reply:
Yeah. I didn't come to this realization for some reason. Converting to `Colormap` happens after length check. But in this case, including a `fake_name` might be redundant?

Reply:
a little redundancy in tests isn't bad :-)